MotionPlanningEnv/sphereObstacle.py

```diff
@@ -87,14 +90,14 @@ def limitHigh(self):
         else:
             return [np.ones(self.dim()) * 1, 1]
 
-    def position(self, **kwargs):
+    def position(self):
         return self._config.geometry.position
 
-    def velocity(self, **kwargs):
-        return np.zeros(self.dim())
+    def velocity(self):
+        raise NotImplementedError
 
-    def acceleration(self, **kwargs):
-        return np.zeros(self.dim())
+    def acceleration(self):
+        raise NotImplementedError
```

Comment:
Why is this change needed?

Reply:
Obstacles could have an initial velocity or acceleration, however, this function now indicates it does not have initial velocity or acceleration. 

I think velocity and acceleration could entirely be skipped unless in the future someone would like initial vel/acc.

Do you want to keep track of vel/acc while the simulation is running? Or mostly for initial vel/acc?